wpeverest/user-registration
# Tweak - Number field validation message override and translatable

### All Submissions:






### Changes proposed in this Pull Request:
The changes in this PR will override jquery validate custom validation messages for the number field with translatable messages.

### How to test the changes in this Pull Request:
Use translation plugins like wpml and check if you can find the validation messages for the number field and try to translate them in other languages.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Number field validation message override and translatable.
